about/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Почему я выбрала направление Front-end разработки</h1>
+    <p>Lorem <strong>ipsum</strong> dolor sit amet consectetur, adipisicing elit. <em>Commodi</em> nulla inventore similique animi perspiciatis voluptate architecto iste doloremque accusamus, illo repudiandae atque, pariatur rerum doloribus! Voluptatem optio minus itaque praesentium.</p>
```

Comment:
If the line became very long, go to the next line.

Reply:
Done.